app/src/main/AndroidManifest.xml

```diff
@@ -10,15 +10,18 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+    
+    
+    
+    
+    <uses-permission android:name="android.permission.MANAGE_USERS" />
 
     <application
```

Comment:
**Reporter**: ANDROIDLINT
**Rule**: AllowBackup
**Severity**: WARN
**File**: C:/Users/anisart/.jenkins/workspace/test3/app/src/main/AndroidManifest.xml L19

AllowBackup/FullBackupContent Problems&#10;On SDK version 23 and up, your app data will be automatically backed up, and restored on app install. Your GCM regid will not work across restores, so you must ensure that it is excluded from the back-up set. Use the attribute &#96;android:fullBackupContent&#96; to specify an &#96;@xml&#96; resource which configures which files to backup. More info: https://developer.android.com/training/backup/autosyncapi.html&#10;The &#96;allowBackup&#96; attribute determines if an application&#39;s data can be backed up and restored. It is documented at http://developer.android.com/reference/android/R.attr.html#allowBackup&amp;#xA;&amp;#xA;By default, this flag is set to &#96;true&#96;. When this flag is set to &#96;true&#96;, application data can be backed up and restored by the user using &#96;adb backup&#96; and &#96;adb restore&#96;.&amp;#xA;&amp;#xA;This may have security consequences for an application. &#96;adb backup&#96; allows users who have enabled USB debugging to copy application data off of the device. Once backed up, all application data can be read by the user. &#96;adb restore&#96; allows creation of application data from a source specified by the user. Following a restore, applications should not assume that the data, file permissions, and directory permissions were created by the application itself.&amp;#xA;&amp;#xA;Setting &#96;allowBackup&#61;&quot;false&quot;&#96; opts an application out of both backup and restore.&amp;#xA;&amp;#xA;To fix this warning, decide whether your application should support backup, and explicitly set &#96;android:allowBackup&#61;(true|false)&quot;&#96;.&amp;#xA;&amp;#xA;If not set to false, and if targeting API 23 or later, lint will also warn that you should set &#96;android:fullBackupContent&#96; to configure auto backup.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1523766183>*